### ISSUE TITLE
confidence_threshold seems to be ignored

### ISSUE THREAD

**Issue by marcreichman** [beniz/deepdetect#239]
I have found that the confidence_threshold option isn't working. It seems from a code review that maybe it would work with bbox enabled but we don't have models handy which do that. Here is an example, using a service running on inception resnet v2 (tf):

```
curl -s -XPOST http://localhost:8081/predict  -d '{
       "service":"inceptionresnetv2_0",
       "parameters":{
         "output":{
           "confidence_threshold":0.9,"best":2
         }
       },
       "data":["http://nationalinterest.org/files/main_images/1024px-USS_New_Jersey_(BB-62)_underway_in_1983.JPEG","https://maxgrace.files.wordpress.com/2009/10/battleship_003.jpg"]
     }' |jq .
{
  "status": {
    "code": 200,
    "msg": "OK"
  },
  "head": {
    "method": "/predict",
    "service": "inceptionresnetv2_0",
    "time": 6632
  },
  "body": {
    "predictions": [
      {
        "uri": "http://nationalinterest.org/files/main_images/1024px-USS_New_Jersey_(BB-62)_underway_in_1983.JPEG",
        "classes": [
          {
            "prob": 0.896259605884552,
            "cat": "n02687172 aircraft carrier, carrier, flattop, attack aircraft carrier"
          },
          {
            "prob": 0.003974397666752338,
            "last": true,
            "cat": "n04347754 submarine, pigboat, sub, U-boat"
          }
        ]
      },
      {
        "uri": "https://maxgrace.files.wordpress.com/2009/10/battleship_003.jpg",
        "classes": [
          {
            "prob": 0.9206534028053284,
            "cat": "n02687172 aircraft carrier, carrier, flattop, attack aircraft carrier"
          },
          {
            "prob": 0.00237444369122386,
            "last": true,
            "cat": "n03216828 dock, dockage, docking facility"
          }
        ]
      }
    ]
  }
}
```

**Comment by beniz** [comment]
Thanks for catching this, fixed by #240.

**Comment by marcreichman** [comment]
Fix is verified, thanks for the quick turnaround!
